### DISCUSSION TITLE
Edge Creation

### DISCUSSION THREAD

**Discussion by baristezel** [eclipse-sirius/sirius-components#3528] [Q&A]
I don't know why, but i couldn't find any example about edge creation from a node. 

I actauly achived it by my previous Aql and Siruis Desktop experince. At least i can creat one to one relations from a node to other node. 
There is two problem. One is my just one of my nodecreations from a node works properly. Second is when i tried to use it on a diagram which i created, whole aligment of the diagram became ridiculous. I have to realign when i use it. You can see my way to achive it below.

![Screenshot from 2024-05-24 16-34-39](https://github.com/eclipse-sirius/sirius-web/assets/9076028/3d1d6edf-1c11-4aab-beeb-49734769c50b)

**Comment by pcdavid** [comment]
Hi,

From the look of your palette (with the "X" at the end), it seems you are using an older version of Sirius Web, which is still based on Sprotty. We switched to React Flow for the diagram rendering, and the issues you see regarding edges routing should be fixed in these more recent versions.

As for the second tool which does not work, it's difficult to know from just the screenshot, but shouldn't the Feature Name be `assemblySystem` (with an upper-case `S`)?
